src/backend/launcher.ts

```diff
@@ -1065,24 +1067,30 @@ async function stopChildProcesses(childPid: number) {
         ])
 
         if (stopResult.stderr) {
-          logError(
-            `Error stopping process with PID ${parentPid}: ${stopResult.stderr}`,
+          if (shouldLog) {
+            logError(
+              `Error stopping process with PID ${parentPid}: ${stopResult.stderr}`,
+              LogPrefix.Backend
+            )
+          }
+        }
+
+        if (shouldLog) {
+          logInfo(
+            `Successfully stopped process with PID: ${parentPid}`,
             LogPrefix.Backend
           )
         }
-
-        logInfo(
-          `Successfully stopped process with PID: ${parentPid}`,
-          LogPrefix.Backend
-        )
       }
 
       stopProcessTree(childPid)
     } catch (error) {
-      logError(
-        `Error executing PowerShell command: ${error}`,
-        LogPrefix.Backend
-      )
+      if (shouldLog) {
+        logError(
```

Comment:
Should we just make these `logDebug`? this will still be pretty spammy when a game is closed and I'm not sure it's necessary

Reply:
Yeah, will do.
they should appear only when aborting from the overlay or the Stop button.